### PR TITLE
Added 404 ignore

### PR DESCRIPTION
**Pull request by MikaHarviala**
Added 404 ignore to handle workspace not found errors

### Description

Mars is seeing errors as Azure returns 'Workspace not found'

### Issues Resolved

POL-386

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/instances_log_analytics_utilization/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.8
+
+- Added 404 ignore to azure instances datasource
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analytics_utilization/CHANGELOG.md:5]: This isn't quite what you changes.  It's ignoring log analytics workspaces that don't exists.  Would you update this to give the customer a better idea of what was changes.  thx

**Review by cdwilhelm — CHANGES_REQUESTED** [review]
a minor change on change log please.

File: cost/azure/instances_log_analytics_utilization/CHANGELOG.md (hunk 1 above)
Comment (MikaHarviala) [reply cost/azure/instances_log_analytics_utilization/CHANGELOG.md]: Updated CHANGELOG to reflect the edit above

**Review by cdwilhelm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**